std/algorithm/comparison.d

```diff
@@ -1382,6 +1385,8 @@ Iterates the passed arguments and returns the minimum value.
 Params: args = The values to select the minimum from. At least two arguments
     must be passed, and they must be comparable with `<`.
 Returns: The minimum of the passed-in values.
+See_Also:
+    $(XREF algorithm,searching,minElement)
```

Comment:
s/XREF/XREF_PACK/, or use the variadic `REF`: `$(REF minElement, algorithm, searching)`